src/modules/bluetooth/config.rs

```diff
@@ -86,14 +86,24 @@ impl Default for FormatConfig {
     }
 }
 
+#[derive(Debug, Deserialize, Clone)]
+#[cfg_attr(feature = "extras", derive(schemars::JsonSchema))]
+#[serde(rename_all = "lowercase")]
+pub enum SizeLimit {
+    Count(i32),
+    Pixel(i32),
```

Comment:
I think `count` is a bit ambiguous (count of what?) - `devices` is a more descriptive name. Pixels should also be plural:

```suggestion
    Devices(i32),
    Pixels(i32),
```